### DISCUSSION TITLE
Support WPF

### DISCUSSION THREAD

**Discussion by vomin0107** [d2phap/ImageGlass#1913] [General]
I tried to use for WPF component. 
But I cannot find how to use imageglass in wpf.
Is there any version of wpf or do you have plan for wpf?

**Comment by d2phap** [comment]
What ImageGlass control do you want to use in WPF?
ImageGlass is an end-user product, not a library control so you will need to copy the source code and build the control by yourself.

### Important note:
ImageGlass is under GPLv3 license, if you intent to use the source code, your app license must be compatible to GPLv3 as well.
